cloudinit/cmd/status.py

```diff
@@ -89,26 +82,21 @@ class StatusDetails(NamedTuple):
 TABULAR_LONG_TMPL = """\
 extended_status: {extended_status}
 boot_status_code: {boot_code}
-{last_update}detail:
-{description}"""
+{last_update}detail: {description}
+errors:{errors}
+recoverable_errors:{recoverable_errors}"""
 
 
 def query_systemctl(
     systemctl_args: List[str],
     *,
     wait: bool,
-    existing_status: Optional[UXAppStatus] = None,
 ) -> str:
     """Query systemd with retries and return output."""
     while True:
         try:
             return subp.subp(["systemctl", *systemctl_args]).stdout.strip()
         except subp.ProcessExecutionError as e:
-            if existing_status and existing_status in (
```

Comment:
After this change, the message to stderr below might get printed once from both `get_status_details()::get_bootstatus()` and again in `get_status_details()::systemd_failed()` when `cloud-init status` is called early in boot. Is this intentional? +1 either way, but it just seems unnecessary from a user's perspective.

Reply:
I refactored it some to account for this. Let me know if that works for you